### PR TITLE
Fix edge case where isSystem() will report erroneus results

### PR DESCRIPTION
**Pull request by jviotti**

### CODE REVIEW

File: lib/linux.coffee [hunk 1]
```diff
@@ -31,4 +31,5 @@ exports.isSystem = (drive, callback) ->
 			return callback(false)
 
 		result = tableParser.parse(stdout)
-		return callback(result[0].RM?[0] isnt '1')
+		rmFlag = result[0].RM?[0] or result[0]['MAJ:MIN'][1]
```
Comment (Page-) [inline lib/linux.coffee]: Why are you looking at the `MAJ:MIN` property for whether it's removable?

Comment (jviotti) [inline lib/linux.coffee]: @Page- There seems to be an edge case where the table parser module I'm using interprets the RM value as the second value of MAJ:MIN. I'll add a comment explaining that.

Comment (Page-) [inline lib/linux.coffee]: Have you/can you report this upstream?

Comment (jviotti) [inline lib/linux.coffee]: @Page- Reported to `table-parser`.

**Pull request merged**